src/interfaces/business-state-interfaces.ts

```diff
@@ -17,13 +17,14 @@ export interface AllowedActionsIF {
   }
 }
 
-/** The Alternate Name (aka operating name) object. */
+/** The Alternate Name object. */
 export interface AlternateNameIF {
   entityType: CorpTypeCd
   identifier: string
+  name: string
   nameRegisteredDate: ApiDateTimeUtc
   nameStartDate: IsoDatePacific
-  operatingName: string
+  nameType: string
```

Comment:
So this is what now comes from Legal API in a business response?

Reply:
Yup, this is a sample:
```json
{
  "business": {
    "alternateNames": [
      {
        "name": "SOME NAME TRANSLATION 1",
        "nameStartDate": "2022-05-01",
        "nameType": "TRANSLATION"
      },
      {
        "name": "SOME NAME TRANSLATION 2",
        "nameStartDate": "2022-05-03",
        "nameType": "TRANSLATION"
      },
      {
        "entityType": "GP",
        "identifier": "FM1000019",
        "name": "FOX FOOD CHAIN",
        "nameRegisteredDate": "2021-04-28T00:00:00+00:00",
        "nameStartDate": "2022-04-29",
        "nameType": "DBA"
      },
      {
        "entityType": "SP",
        "identifier": "FM1111111",
        "name": "SWIFT FOX SNACKS",
        "nameRegisteredDate": "2023-05-10T00:00:00+00:00",
        "nameStartDate": "2023-05-10",
        "nameType": "DBA"
      }
    ],
    "identifier": "FM1000019",
    "legalName": "FOX2 DO2, FOX3 DO3",
    "legalType": "GP"
  }
}
```

Reply:
I recommend creating an enum for the different type values.

Reply:
Even though the UI doesn't nothing with the name type? Hmm...

Reply:
Yeah, use an enum. It's good practicse.

Reply:
Done! I've added the enum so that it looks like the BE: 
![nametype in BE](https://github.com/bcgov/business-filings-ui/assets/122301442/35dad52e-5495-4fe2-98e0-d93de10cc202)

Basically, two values, DBA and TRANSLATION.